code/src/gfx.c

```diff
@@ -377,36 +377,46 @@ static void Gfx_DrawSpoilerItemGroups(void) {
 
 static void Gfx_DrawERTracker(void) {
     if (gSettingsContext.shuffleOverworldEntrances || gSettingsContext.shuffleDungeonEntrances) {
-        u16 itemCount = ENTRANCE_OVERRIDES_MAX_COUNT;
+        u16 itemCount = gEntranceTrackingData.EntrancePairsCount;
         u16 firstItem = entranceScroll + 1;
         u16 lastItem = entranceScroll + MAX_ITEM_LINES;
         if (lastItem > itemCount) { lastItem = itemCount; }
         Draw_DrawFormattedString(10, 10, COLOR_TITLE, "Randomized Entrances (%d - %d) / %d",
             firstItem, lastItem, itemCount);
 
-        // TODO random entrances visited %
+        // TODO random entrances visited percent?
         // Draw_DrawFormattedString(SCREEN_BOT_WIDTH - 10 - (SPACING_X * 6), 10, itemPercent == 100 ? COLOR_GREEN : COLOR_WHITE, "%5.1f%%", itemPercent);

```

Comment:
This is a fun idea. Did you forget to add it? If not, this can be removed.

Reply:
I did forget! Added it in the latest commit.